docs/articles/documentation/reference/configuration-file.md

```diff
@@ -709,6 +710,24 @@ See the command line [--page-load-timeout](command-line-interface.md#--page-load
 *CLI*: [--page-load-timeout](command-line-interface.md#--page-load-timeout-ms)  
 *API*: [runner.run({ pageLoadTimeout })](testcafe-api/runner/run.md)
 
+## browserInitTimeout
+
+Specifies the time (in milliseconds) for every of the specified browsers to connect and report that it is ready to test. If one or more of the browsers fail to connect within the specified period, an error is thrown.
```

Comment:
Perhaps we should clarify where the browsers will connect to (to the TestCafe server)?

Reply:
Is it correct to say that browsers connect to the [TestCafe Object](https://devexpress.github.io/testcafe/documentation/reference/testcafe-api/testcafe/)

Reply:
Yes, I think it's correct.